### DISCUSSION TITLE
Gas Turbine with Heat Recovery Steam Generator Tutorial

### DISCUSSION THREAD

**Discussion by yaqoub12** [oemof/tespy#421] [General]
Hi
Looking for “Gas Turbine with Heat Recovery Steam Generator” tutorial, when do you think will be available?

**Comment by fwitte** [comment]
Dear @yaqoub12,

thank you for you patience and for reaching out, I was in holidays lately, so sorry for that. I have not had much time to pursue on that tutorial, so I unfortunately cannot promise anything. Would you be interested to contribute on it maybe?

Best

**Comment by yaqoub12** [comment]
Thanks Francesco
I woul like to, but i am busy at the moment. I will comeback to you when i
have the time to do it ic not done by then.

Best regards
